develop/parachain-devs/get-started/polkadot-sdk/get-started-psdk.md

```diff
@@ -0,0 +1,137 @@
+
+## Zero to Hero with The Polkadot SDK
+
+The Polkadot SDK is a versatile tool that can be used to build both standalone blockchains and parachains. For standalone blockchains, developers can leverage the SDK's components to create independent networks with custom functionality. The process for deploying a parachain on a Polkadot [core](https://wiki.polkadot.network/docs/learn-agile-coretime#core){target=\_blank} can be summarized as follows:
+
+1. **Reserve** a `ParaId`, where the runtime and genesis state will be uploaded
+2. **Compile** the runtime (written in Rust) to a WebAssembly blob, thereby defining how state transitions from one state to the next. This runtime is created using the Polkadot SDK
+3. **Generate** a chain spec that is viable and ready to be deployed as a live, working parachain
+4. **Generate** the genesis state and wasm
+5. **Obtain** a core through a [Coretime marketplace](https://wiki.polkadot.network/docs/learn-guides-coretime-marketplaces){target=\_blank}
+6. **Assign** that core to a `ParaId`
+7. **Provision** at least one honest, synced collator is available for the *task* to begin producing blocks
+
+!!!info "What is a task?"
+    A [task](https://wiki.polkadot.network/docs/learn-agile-coretime#task){target=\_blank} refers to a process utilizing Polkadot's compute. This could be a parachain or any other computational process, provided that it adheres to the Polkadot protocol.
+
+### Deploying on a Core
+
+After developing the desired runtime and implementing the necessary pallets, the next step is to deploy your parachain on a Polkadot core. This deployment process is crucial as it allows your parachain to leverage the shared security model of the Polkadot network. The deployment process involves several key steps:
+
+```mermaid
+flowchart TD
+    subgraph GA["Generate Artifacts"]
+        direction LR
+        A["Create a runtime"]-->B["Generate a chain specification"]-->C["Generate Genesis State & Runtime Artifacts"]
+    end
+
+    subgraph PC["Procure `ParaId` & Core"]
+        direction LR
+        PARAID["Reserve `ParaId`"]
+        PARAID-->D["Buy Bulk Coretime"]
+        PARAID-->E["Issue On-Demand Coretime Extrinsic"]
+    end
+
+
+    subgraph DEP["Deploying"]
+        direction LR
+        F["Register artifacts to `ParaId`"]-->assign["Assign Core"]-->G["Sync collator"]-->H["Begin producing blocks"]
+    end
+
+GA-->PC
+PC-->DEP
+```
+
+<!-- TODO: We should probably have a link to guides or something here, or build some track which has them go from template to core? -->
+
+<!-- TODO: Add a place / page to install the necessary dependencies -->
+
+### Install Dependencies
+
+Ensure all necessary dependencies are installed.
+
+<!-- TODO: Add a place / page to install the necessary dependencies -->
+
+### Deployment Example - Adder Collator
+
+Try out the preceding example by deploying the adder collator, a very simple "counter" parachain implementation.
+
+## OpenZeppelin Templates & Guides
```

Comment:
I'm not sure if we should include OpenZeppelin Templates as a section because they are already included in the  Polkadot SDK Templates link provided:
![image](https://github.com/user-attachments/assets/ea275bd4-894c-4f2e-ba08-ce10878d1c94)

Reply:
Agreed, will remove and just make it a general templates section.

Reply:
Actually, I will keep it as a sub-section, just so it is covered / mentioned from an SEO standpoint